modules/ROOT/pages/architecture/hexagonale_architecture.adoc

```diff
@@ -0,0 +1,203 @@
+:imagesdir: ../images
+= Hexagonal Architecture
+
+Hexagonal architecture, also known as Ports and Adapters, is a software design pattern that promotes separation of concerns by organizing an application into a central core surrounded by external adapters. 
+The core contains the business logic and communicates with the external world via well-defined ports (interfaces). 
+Adapters implement these ports and handle the translation between the core's domain model and the specific technologies or protocols used by external systems.
+
+[[img-t-hexagonal-architecture]]
+.Hexagonal Architecture Reference
+image::hexagonal_component_architecture_overview.drawio.svg["devonfw hexagonal architecture blueprint",scaledwidth="80%",align="center"]
+// ----
+//Created the directory tree based on this list using https://tree.nathanfriend.io/
+// As the list is easier to maintain, try to do edits in the list structure, use the tool mentioned above and paste both in here:
+
+// - application
+//   - config
+//       - Configuration.java
+//   - core
+//     - domain
+//       - Customer.java
+//       - CustomerFactory.java
+//       - Reservation.java
+//       - Table.java
+//     - ports
+//       - in
+//         - AddReservationPort.java
+//         - CancelReservationPort.java
+//         - AlterReservationPort.java
+//         - AddTablePort.java
+//         - RemoveTablePort.java
+//       - out
+//         - StoreReservationPort.java
+//     - usecase
+//       - AddReservationUc.java
+//       - ManageReservationUc.java
+//       - AddTableUc.java
+//     - [service]
+//       - [FindFreeTableService.java]
+//   - adapter
+//     - in
+//       - rest
+//         - RestController.java
+//         - model
+//          - ReservationDto.java
+//        - mapper
+//          - ReservationMapper.java
+//     - out
+//       - jpa
+//         - JpaAdapter.java
+//         - model
+//           - ReservationEntity.java
+//           - TableEntity.java
+//         - mapper
+//           - ReservationJpaMapper.java
+//           - TableJpaMapper.java
+// ----
+
+[source,plaintext]
+----
+.
+└── application/
+    ├── config/
```

Comment:
This is not explained in detail below. I would recommend to either remove it here or add a short explanation. Is it really important for the understanding of the hexagonal architecture?

Reply:
Removed .config, because I'm not 100% sure if this is a really good idea to always include it. or if the configs (depending of the case) should be more inside the adapters or core